ts/Series/BoxPlot/BoxPlotSeries.ts

```diff
@@ -297,25 +294,62 @@ class BoxPlotSeries extends ColumnSeries {
 
                 // The whiskers
                 if (whiskerLength) {
-                    const whiskerStrokeWidth = point.whiskers.strokeWidth();
-                    highPlot = crisp(point.highPlot, whiskerStrokeWidth);
-                    lowPlot = crisp(point.lowPlot, whiskerStrokeWidth);
-                    pointWiskerLength = (
-                        typeof whiskerLength === 'string' &&
-                        (/%$/).test(whiskerLength)
-                    ) ?
-                        halfWidth * parseFloat(whiskerLength) / 100 :
-                        Number(whiskerLength) / 2;
-                    d = [
-                        // High whisker
-                        ['M', crisp(stemX - pointWiskerLength), highPlot],
-                        ['L', crisp(stemX + pointWiskerLength), highPlot],
-
-                        // Low whisker
-                        ['M', crisp(stemX - pointWiskerLength), lowPlot],
-                        ['L', crisp(stemX + pointWiskerLength), lowPlot]
-                    ];
-                    point.whiskers[verb]({ d });
+                    // Moved here as "const", refactoring of whiskerlength
+                    // triggered "no-loop-func" from eslint
+                    const halfWidth = width / 2,
+                        geWhiskerPair = (
+                            point: BoxPlotPoint
+                        ): SVGPath => {
+                            const strokeWidth = point.whiskers.strokeWidth(),
+                                getWhisker = (
```

Comment:
I would maybe consider drawing these functions out as class methods or static functions. Better for readability, and would make it easier to write unit tests :)